psst-gui/src/webapi/client.rs

```diff
@@ -38,6 +39,43 @@ use crate::{
 
 use super::{cache::WebApiCache, local::LocalTrackManager};
 
+#[derive(Debug, Clone, Data, Lens, Deserialize)]
```

Comment:
Do these structures need to be public? They aren't being called more than once, are they? I think it would make more sense and follow the rest of the code more if we had these in the function that calls the API.